### PR TITLE
Update i18n config to support new Danish translation

### PR DESCRIPTION
**Pull request by kgardnr**
## What does this PR do?

Completes support for Danish.

## How do I test this PR?

- Open Talk
- Change your local storage language to be `da_DK`
- See Talk in Danish!

### CODE REVIEW

File: client/coral-framework/services/i18n.js [hunk 1]
```diff
@@ -6,15 +6,17 @@ import merge from 'lodash/merge';
 import esTA from '../../../node_modules/timeago.js/locales/es';
 import frTA from '../../../node_modules/timeago.js/locales/fr';
 import pt_BRTA from '../../../node_modules/timeago.js/locales/pt_BR';
+import da_DK from '../../../node_modules/timeago.js/locales/da_DK';
```
Comment (alinhle) [inline client/coral-framework/services/i18n.js]: The locales for timeago is called `da`. Also the whole path should be `timeago.js/locales/da` without `../../../node_modules/`. You change it for all others if you are already there :-)

File: client/coral-framework/services/i18n.js [hunk 2]
```diff
@@ -6,15 +6,17 @@ import merge from 'lodash/merge';
...
 import en from '../../../locales/en.yml';
 import es from '../../../locales/es.yml';
 import fr from '../../../locales/fr.yml';
 import pt_BR from '../../../locales/pt_BR.yml';
+import da_DK from '../../../locales/da_DK.yml';
```
Comment (alinhle) [inline client/coral-framework/services/i18n.js]: two imports called `da_DK`

**Review by alinhle — CHANGES_REQUESTED** [review]

File: client/coral-framework/services/i18n.js (hunk 1 above)
Comment (kgardnr) [reply client/coral-framework/services/i18n.js]: Is it `da` or `ta`?

**Comment by kgardnr** [comment]
@cvle take another look!

File: client/coral-framework/services/i18n.js [hunk 3]
```diff
@@ -3,18 +3,20 @@ import has from 'lodash/has';
 import get from 'lodash/get';
 import merge from 'lodash/merge';
 
-import esTA from '../../../node_modules/timeago.js/locales/es';
-import frTA from '../../../node_modules/timeago.js/locales/fr';
-import pt_BRTA from '../../../node_modules/timeago.js/locales/pt_BR';
+import esTA from 'timeago.js/locales/ta'
+import frTA from 'timeago.js/locales/ta';
+import pt_BRTA from 'timeago.js/locales/ta';
+import da_DKTA from 'timeago.js/locales/ta';
```
Comment (alinhle) [inline client/coral-framework/services/i18n.js]: Looks like some copy and paste error, the import paths now all end with `ta` ^^

Comment (alinhle) [reply client/coral-framework/services/i18n.js]: Going with the previous pattern this line should be `import daTA from 'timeago.js/locales/da';`

File: client/coral-framework/services/i18n.js [hunk 4]
```diff
@@ -46,6 +48,7 @@ function init() {
   ta.register('es', esTA);
   ta.register('fr', frTA);
   ta.register('pt_BR', pt_BRTA);
+  ta.register('da_DK', da_DK);
```
Comment (alinhle) [inline client/coral-framework/services/i18n.js]: Going with the previous pattern this line should be ta.register('da_DK', daTA);

**Review by wyattjoh — APPROVED** [review]
Looks good to me!

**Review by alinhle — APPROVED** [review]

**Pull request merged**